README.md

```diff
@@ -533,6 +571,63 @@ project.getMetadata(function(err, metadata) {
 ```
 
 
+## Google Cloud Speech (Limited Preview)
+
+> **This is a Limited Preview release of Google Cloud Speech.** This feature is not covered by any SLA or deprecation policy and may be subject to backward-incompatible changes.
+
+- [API Documentation][gcloud-speech-docs]
+- [Official Documentation][cloud-speech-docs]
+
+```js
+var gcloud = require('gcloud');
+
+// Authorizing on a per-API-basis. You don't need to do this if you auth on a
+// global basis (see Authorization section above).
+
+var speech = gcloud.speech({
+  projectId: 'my-project',
+  keyFilename: '/path/to/keyfile.json'
+});
+
+// Detect the speech in an audio file.
+speech.detect('./audio.raw', {
+  encoding: 'LINEAR16',
+  sampleRate: 16000
```

Comment:
The Speech API does not accept mp3. It accepts only a small subset the many possible audio encodings. It accepts encodings that are typically used with speech.

These are the five encodings that are supported: LINEAR16, FLAC, MULAW, AMR, and AMR_WB

LINEAR16 is raw bytes. The raw audio is not inside any container, and it is uncompressed and lossless. It's the format most commonly accepted by low-level audio APIs.

FLAC is a common lossless compression format. The Speech docs recommend audio be provided in FLAC format.

We _could_ use FLAC as our default, but I wonder if it will confuse users into thinking that speech detection will "just work" on any old audio file. The constraints set forth in the Speech API audio encoding documentation are quite specific, and I think that anyone using the Speech API will _have_ to understand those constraints to be really successful with the Speech API. I'm afraid setting a default here would result in a lot of "it's not working" bugs and support questions, when they just need to read the audio encoding docs to understand exactly what kind of audio they must provide.

On the other hand, requiring that the user provide these two config values forces them to read the audio encoding docs to understand what kind of audio they can provide.

I tried experimenting with aurora.js to auto detect these values, but I had limited success. I can try again.